docs/workflows/create_sync_publish.rst

```diff
@@ -59,6 +59,31 @@ Remote GET response:
 
 .. _versioned-repo-created:
 
+Suse Linux Enterprise Server Remote
+***********************************
+
+If you would like to sync SLES 12+ repository you will need to specify an authentication as ``sles_auth_token``.
+
+You can receive your token with script like this:
+
+.. code:: bash
+
+    curl -H "Authorization: Token token=YourOrganizationRegistrationCode" \
+    https://scc.suse.com/connect/subscriptions/products | \
+    tr "," "\n" | \
+    grep -i "url" | \
+    grep -i "SLE-SERVER"
+
+With your token you can create remote with ``sles_auth_token``.
```

Comment:
This could be slightly clearer to call out the token assumed for the example. Something like:

```
Assuming your token is ``YourRepositoryToken``, create the remote with the ``sles_auth_token`` specified.
```